### PR TITLE
wxmac: add patches for devel

### PR DESCRIPTION
**Pull request by TheZoc**

### CODE REVIEW

**Comment by TheZoc** [comment]
Followed instructions on #1830, I hope this is correct and help!

Cheers!

File: Formula/wxmac.rb [hunk 1]
```diff
@@ -25,6 +25,24 @@ class Wxmac < Formula
...
+      sha256 "c6503ba36a166c031426be4554b033bae5b0d9da6fabd33c10ffbcb8672a0c2d"
+    end
+    
+    # Fix Issue: Building under OS X in C++11 mode for i386 architecture (but not amd64) results in an error about narrowing conversion.
+    patch do
+      url "https://github.com/wxWidgets/wxWidgets/commit/ee486dba32d02c744ae4007940f41a5b24b8c574.patch"
+      sha256 "88ef4c5ec0422d00ae01aff18143216d1e20608f37090be7f18e924c631ab678"
+    end
+    
+        # Fix Issue: Building under OS X in C++11 results in several -Winconsistent-missing-override warnings.
```
Comment (UniqMartin) [inline Formula/wxmac.rb]: The indentation of this comment is wrong. It should be aligned with the following `patch do`.

**Comment by UniqMartin** [comment]
There is some trailing whitespace embedded in your changes. Please run `brew audit --strict --online wxmac` to check for issues and fix them before pushing to the branch of this PR again.

There was also [checklist in our pull request template](https://github.com/Homebrew/homebrew-core/blob/master/.github/PULL_REQUEST_TEMPLATE.md) that we'd like you to complete next time, but maybe you created this PR with a tool like `hub` and thus you sadly never saw our template.

**Comment by TheZoc** [comment]
Hopefully everything is fixed now.
I tried following [this](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#create-your-pull-request-from-a-new-branch), but it screwed up my entire homebrew installation, so I deleted and started from scratch.

Sadly, the instructions aren't too friendly for those who aren't used with git :frowning: 

If anything else needs to be fixed, please, tell me :)

**Comment by UniqMartin** [comment]
Merged in d99f5d5c46afc53cd4207689c6fca73605ee6aa4. Thank you for this contribution to Homebrew, @TheZoc! :tada:

> I tried following [this](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#create-your-pull-request-from-a-new-branch), but it screwed up my entire homebrew installation, so I deleted and started from scratch.
>
> Sadly, the instructions aren't too friendly for those who aren't used with git :frowning: 

I'm sorry to hear this! In theory it should be really difficult to completely screw up, but I agree that Git can be a bit intimidating at first due to its power. And while 99% of mistakes can be fixed without any loss of data, it can be difficult to find out how exactly …

**Pull request merged**